docs/docs/types/message.md

```diff
@@ -0,0 +1,48 @@
+Focused on deliverying outbound communication. Perhaps announcing a product launch, new content, top-level alert, etc. Message modules are simple text modules which can be used in a variety of formats.
+
+``` javascript
+var module = pathfora.Message({
+	// module settings
+});
+
+pathfora.initializeWidgets([ module ]);
+```
+
+
+## layout
+
+Define which layout type the form module should use.
+
+| Value | Type | Behavior |
+|---|---|---|
+| modal | string | module uses a [modal layout](../layouts/modal.md) |
+| slideout | string | module uses a [slideout layout](../layouts/slideout.md) |
+| bar | string | module uses a [bar layout](../layouts/bar.md) |  
+| button | string | module uses a [button layout](../layouts/button.md) |  
```

Comment:
I'm not sure how I feel about this table. At first glance it's neat, but most of the content is either redundant or superfluous. 

An alternative. Take it or leave it:

-----------------------------------

A **string** that defines which layout type the form module will use.

Possible values are
  - `'modal`' (links to the modal layout page)
  - `'slideout'`
  - `'bar'`
  - `'button'`